<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BHAVESH" failures="3" tests="7" name="BankingWebsite" time="70.777" errors="0" timestamp="17 Jun 2022 19:34:38 GMT">
  <testcase classname="com.inetBanking.testCases.TC_LoginTest_001" name="loginTest" time="3.607"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="LoginDataTest" time="3.299">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.LoginDataTest(TC_LoginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataTest -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="LoginDataTest" time="6.892"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="LoginDataTest" time="3.627">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.LoginDataTest(TC_LoginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataTest -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="LoginDataTest" time="3.113">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.LoginDataTest(TC_LoginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataTest -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="LoginDataTest" time="7.397"/>
  <testcase classname="com.inetBanking.testCases.TC_AddCustomer_003" name="AddCustomer" time="10.27"/>
</testsuite> <!-- BankingWebsite -->
